studio/hooks/misc/useNotifications.tsx

```diff
@@ -0,0 +1,18 @@
+import useSWR from 'swr'
+import { Notification } from '@supabase/shared-types/out/notifications'
+
+import { get } from 'lib/common/fetch'
+import { API_URL } from 'lib/constants'
+
+export const useNotifications = () => {
+  const { data, error, mutate } = useSWR<Notification[]>(`${API_URL}/notifications`, get)
```

Comment:
Are the notifications supposed to be filtered once you click into a project? I'm not sure how we want the context shift to work here.

I think it's probably fine for the first pass on this feature, so this is just more me wondering.

Reply:
yeap notifications are at the user level - it'll show notifications for all projects that the user has access to - so no worries about filtering